Bilbary for iPhone/Viiews/StreakView.swift

```diff
@@ -28,20 +28,20 @@ enum ReadGoal: Int, CaseIterable, Codable {
     var label: String {
         switch self {
         case .fiveMinutes:
-            return "5 minutes"
+            return String(localized: "5 minutes")
```

Comment:
Instead of doing that here, can you make `ReadGoal` conform to `CustomStringConvertible` and provide a `description` property and a `localizedDescription`, a bit like that:

```swift
extension ReadGoal: CustomStringConvertible {
    var description: String {
        switch self {
            case .fiveMinutes: "goal.minutes.five"
            ...
        }
    }
    
    var localizedDescription: String {
        String(localized: description)
    }
}
```